mindsdb/integrations/handlers/financial_modeling_prep_handler/__init__.py

```diff
@@ -0,0 +1,19 @@
+from mindsdb.integrations.libs.const import HANDLER_TYPE
+
+from .__about__ import __version__ as version, __description__ as description
+try:
+    from .financial_modeling_handler import FinancialModelingHandler as Handler
+    import_error = None
+except Exception as e:
+    Handler = None
+    import_error = e
+
+title = 'Financial_Modeling'
+name = "financial_modeling"
```

Comment:
I think we should change the name of the integration to `financial_modelling_prep`. financial_modelling seems a little too generic. Let's change the title, example queries and the other aspects of the code to reflect this as well.